packages/suite/e2e/tests/general/check-links.test.ts

```diff
@@ -6,11 +6,17 @@ import routes from '../../../src/constants/suite/routes';
 import { expect, test } from '../../support/fixtures';
 import { createTestAnnotation } from '../../support/reporters/annotations';
 
-async function getAllLinksFromAllPages(page: Page, paths: Array<string>): Promise<Set<string>> {
+async function getAllLinksFromAllPages(
+    page: Page,
+    paths: Array<string>,
+    baseUrl: string,
+): Promise<Set<string>> {
     const allValidHrefs = new Set<string>();
 
     for (const path of paths) {
-        await page.goto(path, { waitUntil: 'domcontentloaded' });
+        const url = baseUrl.endsWith('/') ? baseUrl.slice(0, -1) + path : baseUrl + path;
```

Comment:
```suggestion
        const url = new URL(path, baseUrl).toString();
```

I haven't personally used but this should handle duplicate slashes and is more easily readable

Reply:
Based on Playwright docs, `page.goto()` uses the new URL() constructor to merge the baseURL with the path. As a result, navigating to `./bar.html` will resolve relative to the full path of the baseURL."
So, I've tried to solve this by adding  ```await page.goto(`.${path}`, { waitUntil: 'domcontentloaded' });```